torch/_torch_docs.py

```diff
@@ -2845,6 +2845,36 @@ def merge_dicts(*dicts):
         tensor([False,  True,  False,  True,  False])
 """)
 
+add_docstr(torch.isposinf,
+           r"""
+isposinf(input, out=None) -> Tensor
```

Comment:
isposinf(input, *, out=None) -> Tensor

Reply:
Thanks for the kind advice regarding docs, I have updated the docs accordingly.